docs/articles/sdk/read-system-logs.rst

```diff
@@ -0,0 +1,99 @@
+:orphan:
+
+Reading System Logs with DLT
+============================
+
+Introduction
+------------
+
+PELUX uses a component from GENIVI called `DLT`_ (for Diagnostic Log and Trace)
+which enables distant computers to access the system logs of a machine running
+PELUX.
+
+.. _`DLT`: https://at.projects.genivi.org/wiki/display/PROJ/Diagnostic+Log+and+Trace
+
+DLT is split into two parts:
+
+- **dlt-daemon**: running on PELUX, collecting logs and distributing them on
+  a network
+- **dlt-viewer**: running on a developer machine, accessing distant logs from
+  dlt-daemon
+
+Compiling dlt-viewer
+--------------------
+
+Since dlt-daemon is already present on PELUX, you only need to compile
+dlt-viewer on your development machine. For that, you need to have `git`,
+`cmake` and the Qt SDK installed on your machine.
+
+Then, clone the dlt-viewer repository:
+
+.. code-block:: bash
+
+    git clone https://github.com/GENIVI/dlt-viewer
+
+Compile it with cmake:
+
+.. code-block:: bash
+
+    mkdir build
+    cd build
+    cmake ..
+    make
+
+Running dlt-viewer
+--------------------
+
+You can now run dlt-viewer from its build directory
+
+ .. code-block:: bash
+
+    bin/dlt-viewer
+ 
+Configuring dlt-viewer for your target
+--------------------------------------
+
+dlt-viewer must be configured to point to your PELUX machine. You can do that
+from "*Config -> ECU Add*". On the first tab, named "ECU", feel free to use any
+Id and Description you want and make sure the "Interface Type" is set to TCP.
+On the second tab, named "IP", use the IP address of your target and and make
+sure the port is set to 3490. You can then hit the "OK" button and your machine
+should appear in the "Project" part of the main window.
+
+.. note:: If you don't know the IP address of your target, the `Raspberry Pi
+          documentation`_ describes how to find a new device on the network.
+          This can be used for any target, not just Raspberry Pi
+
+.. _`Raspberry Pi documentation`: https://www.raspberrypi.org/documentation/remote-access/ip-address.md
```

Comment:
We have traditionally listed the links at the end of the document. I'm not sure how it's done in the rest of the blueprint though.